src/test/java/karuna/karuna_backend/content/domain/ContentServiceTest.java

```diff
@@ -22,4 +27,18 @@ void shouldGetContentByPage() {
         assertEquals(Constants.VALUE_PL, contentDTO.contents().get(Constants.KEY));
     }
 
+    @Test
+    void shouldMassUpdateContent() {
+        //given
+        MassContentWrapper massContentWrapper = new MassContentWrapper(List.of(new MassContentDto(Constants.PAGE, Constants.KEY, Constants.VALUE_PL)));
+        //when
+        MassContentWrapper massContentWrapperResponse = contentService.massUpdateContent(massContentWrapper);
+        //then
+        assertEquals(1, massContentWrapperResponse.contents().size());
+        MassContentDto massContentDto = massContentWrapperResponse.contents().get(0);
+        assertEquals(Constants.PAGE, massContentDto.page());
+        assertEquals(Constants.KEY, massContentDto.key());
+        assertEquals(Constants.VALUE_PL, massContentDto.valuePl());
+    }
```

Comment:
By the way I feel that it would be beneficial to add small descriptions after each assert. That is for the future if assert would be a little bit more complicated than those. For now we can leave it like that but I do not see any charm in providing small comments. Up to you

Reply:
Im oponnent of comments in code, #UNCLE_BOB_RULES